docs/UserGuide.md

```diff
@@ -147,7 +147,33 @@ Format : `unmark i/STUDENT_ID DONE_TASK_INDEX`
 Example:
 * `unmark i/AXXXXXXXR 1`
     * Marks the first done task for the student with student ID AXXXXXXXR as undone.
-    
+
+### Editing a student's information: `edit`
+
+Edits a student's information in TAPA.
+
+Format: `edit i/STUDENT_ID n/STUDENT_NAME m/MODULE_CODE [p/PHONE_NUMBER] [h/TELEGRAM_HANDLE] [e/EMAIL_ADDRESS]​`
+
+* The index of the student to be edited is a compulsory field.
+* The student’s matriculation number, name, module code, phone number, telegram handle, and email address fields are optional and can be excluded.
+
```

Comment:
Perhaps we could add what happens if invalid index is given (similar to the commands like "mark" and "unmark")

Reply:
Right, I'll get that done then!

Reply:
@Benjykoh I'll resolve this for now, thanks and let me know if you see any other improvements though :)